knx/router.go

```diff
@@ -144,13 +161,18 @@ func NewRouter(multicastAddress string, config RouterConfig) (*Router, error) {
 // Send transmits a packet.
 func (router *Router) Send(data cemi.Message) error {
 	if data == nil {
-		return errors.New("Nil-pointers are not sendable")
+		return errors.New("nil-pointers are not sendable")
 	}
 
 	// We lock this before doing any sending so the server goroutine can adjust the flow control.
 	router.sendMu.Lock()
 	defer router.sendMu.Unlock()
 
+	// According to the specification, we may choose to always pause for 20 ms
+	// after transmitting, bu we should always pause for at least 5 ms on a multicast address.
+	// Use the safer variant, pause for 20 ms.
+	time.Sleep(20 * time.Millisecond)
```

Comment:
This pause happens before transmitting, not after. Also I don't think it is a good idea to do it. It is entirely pointless by the KNX specification. If somebody needs it, I could agree to an option to enable it.

Reply:
> Any KNX IP device or KNXnet/IP Router SHALL limit the transmission of KNX IP ROUTING_-INDICATION Datagrams to a maximum of 50 Datagrams per second within one second.
For simplicity reasons a KNX IP device MAY choose to always pause 20 ms after it transmits a ROUTING_INDICATION Datagram. A KNX IP device or KNXnet/IP Router SHALL always pause its transmission on an assigned multicast address for at least 5 ms after it transmit a ROUTING_-INDICATION Datagram.

Not only KNXnet/IP Router, but also KNX IP devices. Doesn't that imply that a device should limit its transmission rate to 50 Datagrams per second? Bursting a KNXnet/IP Router will just produce ROUTING_BUSY.

Reply:
That's fine. I'd like this to be optional and when enabled make it comply with the spec by pausing after the transmission not before.